src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Utilities/SpeculationAnalyzer.vb

```diff
@@ -536,8 +536,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Utilities
         End Function
 
         Protected Overrides Function ConversionsAreCompatible(originalExpression As ExpressionSyntax, originalTargetType As ITypeSymbol, newExpression As ExpressionSyntax, newTargetType As ITypeSymbol) As Boolean
-            Dim originalConversion As Conversion?
-            Dim newConversion As Conversion?
+            Dim originalConversion As Conversion? = Nothing
```

Comment:
Which analyzer is flagging this? Wondering why this is not enforced on CI..

Reply:
The error list says BC42108 reported by "Compiler".

Reply:
Is this a false positive from FSA then? Do we know what is causing it to now fire on build, but only in IDE?